src/App.tsx

```diff
@@ -619,6 +619,24 @@ function CreateForm(props: CreateForm) {
     sessionStorage.setItem('draft-role:' + props.draftId, role)
   }
 
+  function handleSubmit(text: string, role: Role) {
+    sessionStorage.removeItem('draft:' + props.draftId)
+    sessionStorage.removeItem('draft-role:' + props.draftId)
+    props.onSubmit(text, role)
+  }
+
+  function handleTextareaKeyDown(e: React.KeyboardEvent<HTMLTextAreaElement>) {
+    const draftText = sessionStorage.getItem('draft:' + props.draftId)
+
+    if (!draftText) {
+      return
```

Comment:
should this be `e.preventDefault()` ?

Reply:
First, thank you for your comment.

My idea to break with `return` is to handle `draftText` when it is null.

I try to implement `e.preventDefault()`, here is result:
![image](https://user-images.githubusercontent.com/6861191/228309339-dd7c5011-7064-45d1-94a5-78c49c9667e2.png)

Should I use `return` as my own code?
Or, if you have any suggestions, please let me know. 😁